packages/voyager-conflicts/src/api/ObjectState.ts

```diff
@@ -16,7 +16,7 @@ export interface ObjectState {
    * @param serverState the data currently on the server
    * @param clientState the data the client wishes to perform some mutation with
    */
-  hasConflict(serverState: ObjectStateData, clientState: ObjectStateData): boolean
+  hasConflict(serverState: ObjectStateData, clientState: ObjectStateData, resolverInfo: any): boolean
```

Comment:
-1 for undocumented parameters that are not clear how they can be handled by implementors.

Reply:
yes I can add some docs

Reply:
added a short doc